entgql/internal/todoplugin/gqlgen.yml

```diff
@@ -32,3 +32,12 @@ models:
   UUID:
     model:
       - entgo.io/contrib/entgql/internal/todouuid/ent/schema/uuidgql.UUID
+  CategoryWhereInput:
+    model:
+      - entgo.io/contrib/entgql/internal/todoplugin/ent.CategoryWhereInput
+  TodoWhereInput:
+    model:
+      - entgo.io/contrib/entgql/internal/todoplugin/ent.TodoWhereInput
+  MasterUserWhereInput:
+    model:
+      - entgo.io/contrib/entgql/internal/todoplugin/ent.MasterUserWhereInput
```

Comment:
Can't you use autobind on the `ent` package like all other examples?

Reply:
On this package, I want to use `@goModel` or map them via the plugin. But, If I add `@goModel` it will break other examples.